locales-src/newtab.ftl

```diff
@@ -185,7 +185,7 @@ newtab-discovery-empty-section-topstories-timed-out = Oops! We almost loaded thi
 # This is shown at the bottom of the trending stories section and precedes a list of links to popular topics.
 newtab-pocket-read-more = Popular Topics:
 newtab-pocket-more-recommendations = More Recommendations
-newtab-pocket-how-it-works = How it works
+newtab-pocket-whats-pocket = What’s Pocket?
 newtab-pocket-cta-button = Get { -pocket-brand-name }
```

Comment:
If the new string is used in the same context as this one, you should be able to use `{ -pocket-brand-name }`